includes/modules/shipping/usps.php

```diff
@@ -434,6 +438,22 @@ protected function checkConfiguration()
         return $this->enabled;
     }
 
+    // -----
+    // Storefront cart check.  Called from storefrontInitialization.  
+    // Will auto-disable the shipping if cart contents cannot be shipped
+    // by USPS
+    //
+    protected function checkCartContents() 
+    { 
+        global $zco_notifier;
+        $contents_ok = true;
+        $zco_notifier->notify('NOTIFY_SHIPPING_CHECK_CART', 'usps', $contents_ok); 
```

Comment:
The USPS shipping method already extends the `base`; no need for a global `$zco_notifier`.  If that 'general' notification isn't going to be documented as part of the Zen Cart base, I'd prefer to have a specific `NOTIFY_USPS_SHIPPING_CART_CHECK` notifier used.

Reply:
It will be part of the Zen Cart base for adding to any shipping method.  But only the plugin methods will use it for a start. 

Removed the global.